transactionoutbox-core/src/test/java/com/gruelbox/transactionoutbox/AbstractDefaultPersistorTest.java

```diff
@@ -33,7 +33,7 @@ abstract class AbstractDefaultPersistorTest {
 
   protected abstract DefaultPersistor persistor();
 
-  protected abstract Dialect dialect();
```

Comment:
There should be no reason for this class and most of the other test classes to change.